device/samples/module_send_event.js

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+'use strict';
+
+require('dotenv').config();
+const { Mqtt } = require('azure-iot-device-mqtt');
+const { clientFromConnectionString : httpClientFromConnectionString } = require('azure-iot-device-http');
+const { ModuleClient, Message } = require('azure-iot-device');
+const { EventHubClient, EventPosition } = require('@azure/event-hubs');
+
+// This is a conceptual example in that it likely does not make sense for a module client to be sending messages then retrieve them in the same script
+// 
+// Here's how the sample works:
+// A Module Client is set up to receive messages from the HTTP Device Client
+// Then, an Event Hub Client, created from our IoT Hub connection string to pull the recent messages down
+
+// Helper Function
+// This returns a dummy messsage ready to be sent, the messageId is just to show order
+const generateImportantMessage = messageId => new Message(
+    JSON.stringify({ 
+        messageId : messageId,
+        data : 'beep boop bopity bop',
```

Comment:
>'beep boop bopity bop' [](start = 15, length = 22)

This doesn't seem like a very important messages.  How about 'Important Message"